tsconfig.json

```diff
@@ -11,6 +11,7 @@
         /* Strict Type-Checking Option */
         "strict": true,   /* enable all strict type-checking options */
         /* Additional Checks */
+        "noImplicitAny": false,
```

Comment:
TypeScript was screaming at me without this

Reply:
In my environment, compiling with typescript doesn't give any warning. So I don't think this change is necessary, but how about it?

Below is a log of running the tsc command on your branch.

```
tsc --version && tsc --noEmit

Version 2.9.2
```

Reply:
Huh, looks like you're right. When I try removing it now, no errors even show up anymore. Not sure what was up with it, I reverted the change in my new commit.